app/src/test/java/org/simple/clinic/cvdrisk/CVDRiskCalculatorTest.kt

```diff
@@ -0,0 +1,89 @@
+package org.simple.clinic.cvdrisk
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.simple.clinic.patient.Answer
+import org.simple.clinic.patient.Gender
+import org.simple.sharedTestCode.TestData
+
+class CVDRiskCalculatorTest {
+
+  private val testData = TestData.cvdRiskCalculationSheet()
```

Comment:
This variable should be `cvdRiskCalculationSheet` instead of `testData`

Reply:
Done